### PR TITLE
Update ktlint version, added new task ktlintFormat 

### PR DESCRIPTION
**Pull request by nsorger**
for applying ktlint to the code, unified build.gradle formatting a tiny bit, formatted code according to the ktlint conventions

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -16,7 +16,7 @@ buildscript {
         classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${kotlinVersion}")
         classpath("org.jetbrains.kotlin:kotlin-allopen:${kotlinVersion}")
         classpath("org.jetbrains.kotlin:kotlin-noarg:${kotlinVersion}")
-        classpath('com.github.ben-manes:gradle-versions-plugin:0.21.0')
+        classpath("com.github.ben-manes:gradle-versions-plugin:0.21.0")
```
Comment (maxkramer) [inline build.gradle]: I noticed that you switched from using single quotes to double quotes in this file. It was originally single quotes for everything since double quotes are only needed for templating variables. Why was this changed?

```suggestion
        classpath('com.github.ben-manes:gradle-versions-plugin:0.21.0')
```

**Review by maxkramer — CHANGES_REQUESTED** [review]
Thanks for the update, one comment!

File: build.gradle (hunk 1 above)
Comment (nsorger) [reply build.gradle]: I just noticed the inconsistent use of single and double quotes. Didn't occur to me that it is related to variable substitution. I can revert that change if you want.

**Review by maxkramer — APPROVED** [review]
